specification/network/resource-manager/readme.ruby.md

```diff
@@ -13,6 +13,7 @@ azure-arm: true
 ``` yaml $(ruby) && $(multiapi)
 batch:
   - tag: package-2018-12-only
+  - tag: package-2018-11-only
```

Comment:
This tag is not defined in the readme.md file. Did you miss to add the changes?

Reply:
Yeah, that was missed. Added now.